tests/acceptance/tuples.manifold

```diff
@@ -0,0 +1,17 @@
+xIn1 = primitive port Bool;
+xIn2 = primitive port Int;
+xOut = primitive port Int;
+xIn1Pin = primitive node (Nil) -> (x: xIn1);
+xIn2Pin = primitive node (Nil) -> (x: xIn2);
+xOutputPin = primitive node (x: xOut) -> (Nil);
+
+cowFn = (isCow: Bool, grassEaten: Int) -> (outIsCow: Bool, methaneEmissions: Int) {
+  outIsCow = isCow;
+  methaneEmissions = grassEaten;
+};
+
+a = xIn1Pin();
+b = xIn2Pin();
+thisIsACommentICannotAttachBelowToAnOutput = cowFn(p: a, q: b);
+thisIsACommentICannotAttachBelowToAnOutput.methaneEmissions;
+thisIsACommentICannotAttachBelowToAnOutput[1];
```

Comment:
In this test, the graph for thisIsACommentICannotAttachBelowToAnOutput gets correctly made as best I can tell, but it's pruned from the below output because I can't connect anything from a tuple output to a primitive port.